waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -76,6 +70,12 @@ deleteGeneratedMigrationsDirIfRedundant wasp projectRootDir = do
   where
     projectMigrationsDirAbsFilePath = SP.fromAbsDir $ projectRootDir </> dbRootDirInProjectRootDir </> dbMigrationsDirInDbRootDir
 
+-- * Db generator
```

Comment:
That was already there (comment above). Should we nix any, all 3 of them?

Reply:
Aha, ok in that case we can leave them. In general I tend to push for removing comments that repeat what the code says, since they introduce redundancy and redundancy has to be maintained to not go out of sync. Rule I use when I see a comment is to ask myself is there is any way that could be captured in the code, so that comment can be removed. But that said, sometimes they are useful for faster reading / overview, so it is better not to be too strict and be flexible. Anyway, whatever you like better, if you feel it helps a bit, let's leave them!

Reply:
Yeah I agree they are probably not needed and we don't lose anything by removing all of them. Will do 👍🏻